bindings/pyroot/pythonizations/python/ROOT/pythonization/_roofit/_roodataset.py

```diff
@@ -54,3 +54,200 @@ def plotOnXY(self, *args, **kwargs):
         # Redefinition of `RooDataSet.plotOnXY` for keyword arguments.
         args, kwargs = _kwargs_to_roocmdargs(*args, **kwargs)
         return self._plotOnXY(*args, **kwargs)
+
+    @staticmethod
+    def from_numpy(data, variables, name=None, title=None, weight_name=None, clip_to_limits=True):
+        """Create a RooDataSet from a dictionary of numpy arrays.
+        Args:
+            data (dict): Dictionary with strings as keys and numpy arrays as
+                         values, to be imported into the RooDataSet.
+            variables (RooArgSet): Specification of the variables in the
+                                   RooDataSet, will be forwarded to the
+                                   RooDataSet constructor. Both real values and
+                                   categories are supported.
```

Comment:
Does this accept lists?

Reply:
That's true, I will specify that in the docs.